package.json

```diff
@@ -15,27 +15,41 @@
   },
   "dependencies": {
     "ajv": "^6.4.0",
-    "chalk": "^2.4.0",
+    "body-parser": "^1.18.2",
+    "chalk": "^2.3.2",
     "commander": "^2.15.1",
+    "express": "^4.16.3",
     "fs-extra": "^5.0.0",
     "listr": "^0.14.0",
     "lodash": "^4.17.5",
     "pjson": "^1.0.9",
+    "request": "^2.85.0",
+    "request-promise": "^4.2.2",
+    "rimraf": "^2.6.2",
     "which": "^1.3.0",
+    "winston": "^2.4.1",
     "yamljs": "^0.3.0",
     "zen-observable": "^0.8.8"
   },
   "devDependencies": {
+    "@types/express": "^4.11.1",
     "@types/fs-extra": "^5.0.2",
+    "@types/get-port": "^3.2.0",
     "@types/listr": "^0.13.0",
-    "@types/lodash": "^4.14.107",
+    "@types/lodash": "^4.14.106",
     "@types/node": "^9.6.5",
+    "@types/mkdirp": "^0.5.2",
+    "@types/request": "^2.47.0",
+    "@types/request-promise": "^4.1.41",
+    "@types/rimraf": "^2.0.2",
     "@types/which": "^1.3.1",
+    "@types/winston": "^2.3.9",
     "@types/yamljs": "^0.2.30",
     "ava": "^1.0.0-beta.3",
     "coveralls": "^3.0.0",
-    "nyc": "^11.7.1",
-    "rimraf": "^2.6.2",
+    "install": "^0.11.0",
+    "npm": "^5.8.0",
```

Comment:
no need for npm. what does "install" do? i think you don't need that one either.